ci-operator/step-registry/assisted/nutanix/setup/configure/assisted-nutanix-setup-configure-commands.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ nutanix assisted test-infra setup machine command ************"
+
+# ensure LEASED_RESOURCE is set
+if [[ -z "${LEASED_RESOURCE}" ]]; then
+  echo "Failed to acquire lease"
+  exit 1
+fi
+
+# shellcheck source=/dev/random
+source $SHARED_DIR/nutanix_context.sh
+
+echo "$(date -u --rfc-3339=seconds) - Getting PE Name"
+
+pc_url="https://${PE_HOST}:${PE_PORT}"
+clusters_api_ep="${pc_url}/api/nutanix/v3/clusters/list"
+un="${NUTANIX_USERNAME}"
+pw="${NUTANIX_PASSWORD}"
```

Comment:
Not a must for this PR, just wanted to see if it's possible to be changed afterwards

Reply:
Seems Nutanix supports [SAML, AD and local auth](https://portal.nutanix.com/page/documents/details?targetId=Nutanix-Security-Guide-v6_0%3Amul-security-authentication-pc-iam-c.html), and the only option for local users is username/password. It comes from cluster profile, so the change is possible but not likely (since its apparently a part of install-config now?)